src/System.Private.CoreLib/src/System/Runtime/InteropServices/ComEventsMethod.cs

```diff
@@ -123,24 +121,25 @@ public static ComEventsMethod Find(ComEventsMethod methods, int dispid)
             return methods;
         }
 
-        public static ComEventsMethod Add(ComEventsMethod methods, ComEventsMethod method)
+        public static ComEventsMethod Add(ComEventsMethod? methods, ComEventsMethod method)
         {
             method._next = methods;
             return method;
         }
 
-        public static ComEventsMethod Remove(ComEventsMethod methods, ComEventsMethod method)
+        public static ComEventsMethod? Remove(ComEventsMethod methods, ComEventsMethod method)
         {
             Debug.Assert(methods != null, "removing method from empty methods collection");
             Debug.Assert(method != null, "specify method is null");
 
             if (methods == method)
             {
-                methods = methods._next;
+                return methods._next;
```

Comment:
@AaronRobinsonMSFT, I made a few implementation tweaks here, as it seemed like the best way to satisfy the compiler while still keeping the signatures the way I presumed we wanted them.  Mind taking a look?